unique.c

```diff
@@ -0,0 +1,31 @@
+/*Jinoo Hong*/
+/*Implement an algorithm to determine if a string has all unique characters. Name this program `unique.c`.*/
+
+#include <stdio.h>
```

Comment:
Perfect job! The only thing is I would recommend more comments to explain your design decisions!